### DISCUSSION TITLE
Feedback: Design Area Dynamic Sidebar Feature Preview

### DISCUSSION THREAD

**Discussion by fredmaggiowski** [mia-platform/community#212] [Product Feedback]
# Design Area Dynamic Sidebar Feature Preview

With Console SaaS v10.6.0 there is a new sidebar available in preview, what do you think about it?

Give us your feedback here!

## Feature description

> By enabling the Dynamic Sidebar of the Design area, you will see at the top only the initial sections that you are actually using or for which there is a dependency with the resources you have configured. 
> All the remaining and non-used sections will initially be grouped under the expandable section "More Features".
>
> As soon as you start using resources linked to these sections, they will automatically be taken out of the "More Features" section, according to the dependencies identified in your configuration.

![e2700009-c8a0-4b17-8f5d-1c5fa63d5677(1)](https://user-images.githubusercontent.com/7142570/225705340-860c5e13-b4e2-4e3d-8672-28196233c25b.png)

## How to opt-in?

The feature is opt-in, therefore you have to activate it for yourself! Head to your avatar image and select the **Feature Preview** tab, [check out the documentation to know more](https://docs.mia-platform.eu/docs/release-notes/v10.6.0#feature-preview-section-only-available-for-saas-console-users)!

**Comment by peppemanzi** [comment]
I really like this feature. It allows you to focus only on the useful sections and it is clearer how the sections relate to the microservices in your project.

**Comment by danibix95** [comment]
Hi, I like this new feature! However, activating it when using the sidebar collapsed makes the non-configured sections disappearing.
In order to enter one of the _more features_ section I need to first expand the sidebar and only then I can select the section of _more features_ I'm interested in.

Would it be possible to provide a way to show the sections under _more features_ without the need of expanding the sidebar?

**Reply by fredmaggiowski** [reply]
Hi @danibix95 thank for your feedback, the main purpose of the sidebar is to actually let you focus on what you are using without the noise provided by unused features; therefore the presence of the *More Features* collapsed menu. We experimented with simple feature sorting but it was harder to interact with.

We could understand how to add the *More Feature* icon to the collapsed sidebar with something like a popup menu to allow you to interact with it without expanding the sidebar. @albertotessarotto FYI

**Comment by nicola88** [comment]
Closing since feature is GA.
